tasks/sanity-inspect-image.yaml

```diff
@@ -49,3 +51,5 @@ spec:
       echo "The base image is $BASE_IMAGE, get its manifest now"
       skopeo inspect --no-tags docker://$BASE_IMAGE  > $BASE_IMAGE_INSPECT || true
       echo "$BASE_IMAGE" | tee $(results.BASE_IMAGE.path)
+
+      jq -r ".Name" $BASE_IMAGE_INSPECT | cut -d"/" -f2,3 | tee $(results.BASE_IMAGE_REPOSITORY.path)
```

Comment:
If we don't make sure the registry is always `registry.access.redhat.com`, so should we also expose the registry to result? Otherwise the `IMAGE_REGISTRY` in deprecated-image-check.yaml is useless.

Reply:
Actually `BASE_IMAGE` is like `quay.io/hongweiliu/test_image:tag`, you may remove `BASE_IMAGE` and expose base image registry as well. Thanks!

Reply:
There is an issue with the entries for Red Hat images - in Pyxis, they are always defined as `registry.access.redhat.com`, even though the image could be hosted in `registry.redhat.io` or similar. 
Another issue is that Pyxis contains only the entries to Red Hat images and nothing else. This is why the plan is to attempt to test the image repository _as if_ it was hosted in `registry.access.redhat.com`. If we don't find the entry in Pyxis, we will exit gracefully from the task.

Reply:
I see now. Thanks.